static/app/utils/profiling/gl/utils.ts

```diff
@@ -198,13 +198,13 @@ export class Rect {
     return this.x;
   }
   get right(): number {
-    return this.left + this.width;
+    return this.x + this.width;
```

Comment:
Is there a reason to prefer `this.x` over `this.left`? Also if they refer to the same value, can we consolidate this and strictly use `this.left` and that's more explicit compared to `this.x`.

Reply:
Yeah, you are right. Left is mostly here because we also have "right", so left is basically a duplicate of x and right is the useful one. I think I originally added it just because it made sense to cover all the coordinates as I added the right and bottom method, but you are right that it creates confusion. I still find it semi-useful in a ways because I would use them differently given the context/operation that I'm doing. E.g. for draw calls, I rather use left/right, as they imply some computed value whereas x, y I would mostly use for more "math" like operations.